jvm/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -458,6 +458,23 @@ object ConverterUtils extends Logging {
   final val EXP = "exp"
   final val POWER = "power"
   final val PMOD = "pmod"
+  final val ROUND = "round"
+  final val GREATEST = "greatest"
+  final val LEAST = "least"
+
+  // PrestoSQL Math fucctions of Velox
+  final val ACOS = "acos"
+  final val ASIN = "asin"
+  final val ATAN = "atan"
+  final val ATAN2 = "atan2"
+  final val COS = "cos"
+  final val COSH = "cosh"
+  final val DEGREES = "degrees"
+  final val LOG10 = "log10"
```

Comment:
This is a duplicated pr for these functions, please re-check. #520

Reply:
Seen.
Only one day apart.
Will review each other.

Reply:
LGTM